src/aggregation/bucket/term_agg.rs

```diff
@@ -81,6 +81,7 @@ pub struct TermsAggregation {
     ///
     /// Should never be smaller than size.
     #[serde(skip_serializing_if = "Option::is_none", default)]
+    #[serde(alias = "split_size")]
```

Comment:
nitpick... can we do it the other way around?

variable named split_size and alias = "shard_size" ?

Reply:
yep, done